contracts/governance/airdrop.cairo

```diff
@@ -0,0 +1,2463 @@
+%lang starknet
+
+@storage_var
+func airdrop_claimed(claimee: Address) -> (res: felt) {
+}
+
+@external
+func claim{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(claimee: Address) {
+    alloc_locals;
+
+    let (governance_token_addr) = governance_token_address.read();
```

Comment:
I don't see the governance_token_address storage var

not sure that it is caused by this crazy long file, or because its missing

Reply:
It's in `upgrades.cairo` , protostar automatically concats all the files